src/jit/morph.cpp

```diff
@@ -3759,14 +3769,19 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
                     }
                 }
 
-#ifndef _TARGET_X86_
-                // TODO-Arm: Does this apply for _TARGET_ARM_, where structs passed by value can be split between
-                // registers and stack?
+#ifdef _TARGET_64BIT_
                 if (size > 1)
                 {
                     hasMultiregStructArgs = true;
                 }
-#endif // !_TARGET_X86_
+#elif defined(_TARGET_ARM_)
+                // TODO-Arm: Where structs passed by value can be split between
+                // registers and stack?
```

Comment:
Please phrase this either as a complete question or statment, e.g. "TODO-Arm: Handle the case where structs passed by value can be split between registers and stack."